src/firebase/api/gcp-utils.js

```diff
@@ -1,74 +1,82 @@
-import { storage, pubsub as gcpPubsub } from '../firebase.js';
-import { pubsub } from '../constants.js';
+import { app, pubsub as gcpPubsub } from '../firebase.js';
+import { pubsub, storageBuckets } from '../constants.js';
 
 /**
  * Upload an image to the specified bucket
  * @param {string} userId userId
  * @param {string} experimentId experimentId
  * @param {Blob} image image file to be uploaded
- * @param {string} bucket destination bucket
  *
  * @return {JSON}
  */
 export const uploadImageToStorage = async (
-  userId, experimentId, image, bucket) => {
+  userId, experimentId, image) => {
   const imagePath = `${userId}-${experimentId}/${image.name}`;
 
-  const storageRef = storage.ref();
-  const rawImageBucketRef = storageRef.child(bucket);
+  const rawImageBucketRef = app.storage(storageBuckets.SIE_RAW_IMGS).ref();
```

Comment:
does `storage.child(storageBuckets.SIE_RAW_IMGS)` work?

Reply:
No, if we call firebase.storage() it will only refer to the default bucket, that's why we can see a sie-raw-images folder inside the default bucket now in the console instead of in the sie-raw-images bucket. In order to access the customized bucket, we need to use firebase.app().storage() according to the documentation.

Reply:
TIL! Thank you!